components/SensorComponent.js

```diff
@@ -55,42 +89,46 @@ export const SensorComponent = () => {
       Accelerometer.addListener((accelerometerData) => {
         // Whenever this function is called, we have received new data
         // The frequency of this function is controlled by setUpdateInterval
-        setData(accelerometerData);
+        setData(accelerometerData)
       })
-    );
-  };
+    )
+  }
 
   // This will tell the device to stop reading Accelerometer data.
   // If we don't do this our device will become slow and drain a lot of battery
   const _unsubscribe = () => {
-    subscription && subscription.remove();
-    setSubscription(null);
-  };
+    subscription && subscription.remove()
+    setSubscription(null)
+  }
 
   useEffect(() => {
     // Start listening to the data when this SensorComponent is active
-    _subscribe();
+    _subscribe()
 
     // Stop listening to the data when we leave SensorComponent
-    return () => _unsubscribe();
-  }, []);
+    return () => _unsubscribe()
+  }, [])
+
+  useEffect(() => {
+    if (isShaking(data)) {
+      getRandomJoke()
+    }
+  }, [data])
+
+  let [fontsLoaded] = useFonts({
+    Bangers_400Regular,
+  })
+
+  if (!fontsLoaded) {
+    return <AppLoading />
+  }
 
   return (
     <ShakeView>
-      {/* 
-      If isShaking returns true:
-        - We could render conditionally
-        - Maybe we want to dispatch some redux event when device shakes?
-        - Maybe change some styled props? 
-      */}
-      {isShaking(data) && <ShakeAlert>Shaking</ShakeAlert>}
-      <ShakeDataView>
-        <ShakeDataTitle>Shake Data</ShakeDataTitle>
-        {/* toFixed(2) only shows two decimal places, otherwise it's quite a lot */}
-        <ShakeData>X: {data.x.toFixed(2)}</ShakeData>
-        <ShakeData>Y: {data.y.toFixed(2)}</ShakeData>
-        <ShakeData>Z: {data.z.toFixed(2)}</ShakeData>
-      </ShakeDataView>
+      <Title style={{ fontFamily: 'Bangers_400Regular', fontSize: 40 }}>
```

Comment:
Really nice job with the fonts! I tried this for ages but never managed to get it to work properly!